libs/cardano-data/testlib/Test/Cardano/Data/Arbitrary.hs

```diff
@@ -1,10 +1,16 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE ImportQualifiedPost #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
 
 module Test.Cardano.Data.Arbitrary where
 
-import Data.OSet.Strict
+import Data.OMap.Strict qualified as OMap
+import Data.OSet.Strict qualified as OSet
 import Test.Cardano.Ledger.Binary.Arbitrary ()
 import Test.QuickCheck
 
-instance (Arbitrary a, Ord a) => Arbitrary (OSet a) where
-  arbitrary = fromSet <$> arbitrary
+instance (Arbitrary a, Ord a) => Arbitrary (OSet.OSet a) where
+  arbitrary = OSet.fromSet <$> arbitrary
```

Comment:
I just realized that relying directly on the Set will make the values always appear in a sorted order, which is not good for testing.
```suggestion
  arbitrary = OSet.fromSet . shuffle . Set.toList <$> arbitrary
```